### PR TITLE
docs(contributing): update commands to start storyboard

### PR DESCRIPTION
**Pull request by rachidBensaid**
## Summary

Add missing command `pnpm build` before running `pnpm run start`, otherwise we get an error when we run it for the first time

## Type

- Documentation

#### What is expected?

The missing command in the `contributing` file

#### The following changes where made:

1. Updated contributing.md

### CODE REVIEW

**Review by matthprost — APPROVED** [review]

File: .changeset/metal-dogs-fail.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+"@ultraviolet/form": patch
+"@ultraviolet/icons": patch
+"@ultraviolet/illustrations": patch
+"@ultraviolet/plus": patch
+"@ultraviolet/themes": patch
+"@ultraviolet/ui": patch
+---
+
+Update 'contributing.md' to add a necessary command during the installation
```
Comment (matthprost) [inline .changeset/metal-dogs-fail.md:10]: We don't do changeset for non-functional changes (documentation, testing, etc)

**Review by matthprost — CHANGES_REQUESTED** [review]

**Review by alexandre-combemorel — APPROVED** [review]

**Review by vincentaudebert — APPROVED** [review]
other than changeset, all good

**Review by DorianMaliszewski — APPROVED** [review]

File: .changeset/metal-dogs-fail.md (hunk 1 above)
Comment (rachidBensaid) [reply .changeset/metal-dogs-fail.md:10]: ok, i did it because i had a message saying that the changeset was missing, i'll remove it

Comment (matthprost) [reply .changeset/metal-dogs-fail.md:10]: No worries, thanks for the contribution!

**Comment by matthprost** [comment]
I merged your change with another PR as I cannot push into this one: https://github.com/scaleway/ultraviolet/pull/3609

**Pull request closed without merging**